decidim-proposals/app/cells/decidim/proposals/coauthorships_cell.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # This cell renders a collapsible list of the proposal coauthors.
+    #
+    # Available sizes:
+    #  - `:small` => collapses after 3 elements.
+    #  - `:default` => collapses after 7 elements. If not specified, this one is
+    #    used.
+    #
+    # Example:
+    #
+    #    cell("decidim/proposals/coauthorships", @proposal)
+    class CoauthorshipsCell < Decidim::ViewModel
+      include Decidim::ApplicationHelper
+
+      def show
+        if proposal.official?
+          cell "decidim/author", present(proposal).author, from: proposal
+        else
+          cell(
+            "decidim/collapsible_authors",
+            authors_for(proposal),
+            cell_name: "decidim/author",
+            cell_options: { extra_classes: ["author-data--small"] },
+            size: :small,
+            from: proposal
+          )
+        end
+      end
+
+      private
+
+      def proposal
```

Comment:
I don't see the point of this method, as `model` works fine and is more abstract, since in the near future `coauthorships_cell` will be used also in `collaborative_drafts` also.
So I would `s/proposal/model`